auto-self-nominate/index.ts

```diff
@@ -0,0 +1,169 @@
+import { U64 } from "codechain-primitives/lib";
+import { SDK } from "codechain-sdk";
+import { PlatformAddress } from "codechain-sdk/lib/core/classes";
+import { Custom } from "codechain-sdk/lib/core/transaction/Custom";
+
+const RLP = require("rlp");
+
+const ACTION_TAG_SELF_NOMINATE = 4;
+const STAKE_ACTION_HANDLER_ID = 2;
+const NEED_NOMINATION_UNDER_TERM_LEFT = 5;
```

Comment:
How about making it configurable with the default value?

Reply:
And I think 2 is enough for the default value.

Reply:
made it configurable with the default value 2